rfcs/text/0000-Incident-Response-Plan.md

```diff
@@ -30,23 +30,169 @@ As DCP governance, I want be notified of a confirmed or suspected incident so th
+* Establish an incident response capability
+* Develop incident response policies and procedures
+
+##### Phase 2: Detection and Analysis
+Incident identification occurs in this phase. The DCP IRT learns of an actual or suspected security incident. Incidents may be detected through many different means with varying levels of detail and veracity. Security incidents may be detected through some of the following examples:
+* End user identification
+* System administrator identification
+* Log review
+* Automated alerting tools
+* Locked out accounts
+* Pop-ups and redirected websites when browsing 
+* Unexpected software installs
+* Unexplained changes to files
+* Anomalies in normal network traffic patterns
+* Abnormal outbound traffic
+* Irregular access locations
+* Large number of requests for the same object or files
+* Suspicious activity on the network after-hours
+* Multiple failed login attempts
+* Unknown/unauthorized IP addresses
+* Services and applications configured to launch automatically
+
+If there is confirmation of an incident, the IRT formulates an overall response strategy and reports the incident to the Oversight Committee and other concerned parties, as appropriate.
+
+##### Phase 3: Containment, Eradication, and Recovery
+This phase is used to gather electronic and physical evidence to determine exactly what has happened, who might be responsible, and how this type of intrusion can be prevented in the future.  The DCP IRT will make recommendations to remediate affected resources and verify completion. As necessary, the IRT will take measures to limit the amount of damage that an intruder or authorized affected system can inflict upon the rest of the IT infrastructure. 
+
+##### Phase 4: Post-Incident Activity
+Follow-up occurs in this phase. DCP IRT completes a report of the incident, subsequent recovery process, and lessons learned. A final copy of the incident report is sent to the Oversight Committee and appropriate stakeholders. Additionally, an incident post-mortem may be conducted to evaluate the lessons learned during the event.
+
+#### Incident Response Procedures
+Anyone can declare an incident. Follow the procedure outlined below if you suspect or have confirmed that an incident has occurred.
+
+__Step 1__: Personnel identifying the suspected or confirmed incident (Incident Handler) will alert the DCP IRT by submitting a ticket to the to-be-created humancellatlas/dcp-security private GitHub repo. The ticket will include the following information to ensure appropriate triage:
+1. Assign the ticket to the DCP IRT Coordinator
+1. Date of discovery of the suspected or confirmed incident
+1. Brief description of suspected or confirmed incident
+1. If known, also include:
+   1. Date incident occurred
```

Comment:
I have seen before where the reporter can classify the ticket initially (event, incident, etc) and then the reviewer can downgrade/change as needed. In this situation, are all defaulting to a certain severity?

Reply:
I've updated to specifically called this out within today's updates